tests/phpunit/tests/theme/wpGetGlobalStylesheet.php

```diff
@@ -26,6 +26,9 @@ class Tests_Theme_WpGetGlobalStylesheet extends WP_Theme_UnitTestCase {
 	private $switch_to_default_theme_at_teardown = false;
 
 	public function tear_down() {
+		// Reset development mode after each test.
+		$GLOBALS['_wp_tests_development_mode'] = '';
```

Comment:
Better:
```suggestion
		unset( $GLOBALS['_wp_tests_development_mode'] );
```

The production code checks for this rely on `isset()`, not on `! empty()`, so we should rather unset any such global that may have been set during tests (as the default is unset too).

Reply:
Thanks for the suggestion. Both seem to fix the Unit test issue, but happy to apply these changes. Done in [f83c7b2](https://github.com/WordPress/wordpress-develop/pull/5533/commits/f83c7b23a659725fe64606a6270f55286882db66). In the future we should consider moving all of these types of things to a test utility function.